galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -371,3 +371,18 @@ messages:
     args:
       - name: annotation
         type: string
+
+  - name: "DeploymentConflictingPorts"
+    code: IST0137
+    level: Warning
+    description: "If we have two services, both selecting the same workload, with the same target port, they should be the same port."
```

Comment:
Reword without "if"?  Perhaps "Two services selecting the same workload with same target port are MUST refer to the same port."